ddl/ddl_worker.go

```diff
@@ -188,7 +188,7 @@ func (d *ddl) handleDDLJobQueue() error {
 				return errors.Trace(err)
 			}
 
-			if job.IsRunning() {
+			if job.IsRunning() || job.IsSynced() {
```

Comment:
We want to make sure the job waits for 2*lease or finds all servers' schema version are the latest version. You can read the comments of `JobSynced`.